test_suite/slice.sugar.jsonnet

```diff
@@ -0,0 +1,106 @@
+local arr = std.range(0, 5);
+local str = "012345";
+local a = 2;
+local b = 4;
+
+local arrCases = [
+    {
+        input:  arr[2:4],
+        output: [ 2, 3 ],
+    },
+    {
+        input:  arr[2:4:1],
+        output: [ 2, 3 ],
+    },
+    {
+        input:  arr[:4],
+        output: [ 0, 1, 2, 3 ],
+    },
+    {
+        input:  arr[2:],
+        output: [ 2, 3, 4, 5 ],
+    },
+    {
+        input:  arr[:],
+        output: arr,
+    },
+    {
+        input:  arr[:1000],
+        output: arr,
+    },
+    //{
+    //    input:  arr[::],
+    //    output: arr,
+    //},
```

Comment:
I guess you can change the parser to check for that, a bit like C++ does with C<vector<int>>